File: pertemuan1/tiket01.py
kursi_tersedia = 50
pesanan = int(input("Masukkan jumlah kursi yang ingin dipesan: "))
  
if pesanan <= kursi_tersedia:
    kursi_tersedia -= pesanan
    print(f"Pemesanan berhasil! Sisa kursi: {kursi_tersedia}")
else:
    print("Kursi tidak cukup.")

File: pertemuan1/tiket01.js
let kursiTersedia = 50;
let pesanan = parseInt(prompt("Masukkan jumlah kursi yang ingin dipesan: "));

if (pesanan <= kursiTersedia) {
  kursiTersedia -= pesanan;
  console.log(`Pemesanan berhasil! Sisa kursi: ${kursiTersedia}`);
} else {
  console.log("Kursi tidak cukup.");
}
